src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+} from './ChartFooter.style';
+import { useBreakpoint } from 'common/hooks';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string | null;
+  shareButton: any;
+  aboutModal?: AboutModalProps;
```

Comment:
_i think_ all of the footers will have an info modal, so we can make this `aboutModal` prop required and nix the `?`